docs/src/routes.tsx

```diff
@@ -33,11 +33,25 @@ import PopupsPrototype from './prototypes/popups'
 import IconViewerPrototype from './prototypes/IconViewer'
 import MenuButtonPrototype from './prototypes/MenuButton'
 import AlertsPrototype from './prototypes/alerts'
+import { Routes as BrowserTestRoutes } from '../../e2e'
+import { match } from 'react-router'
+
+type E2EProps = {
+  match: match<{
+    exampleName: string
+  }>
+}
+
+const E2EExample: React.FC<E2EProps> = ({ match }) => {
+  const Component = BrowserTestRoutes[match.params.exampleName]
+  return <Component />
+}
 
 const Router = () => (
   <BrowserRouter basename={__BASENAME__}>
     <Switch>
       <Route exact path="/maximize/:exampleName/:rtl?" component={ExternalExampleLayout} />
+      <Route exact path="/e2e/:exampleName" component={E2EExample} />
```

Comment:
I am not fully support the idea that we injecting this to existing docsite because it's clearly separated, similar case to perf examples. Can we use a separate config to bundle this?

Reply:
currently some of the tests are running against doc site examples - while some of them will run against custom ones, thus. it is necessary to run doc site, as we are doing it for other visual/perf tests. 

Lets defer it to future PR - currently we are not making things worse, as there are some artifacts in docs that we are using for perf tests, as an example. I've just made these e2e tests being non-distractive to the client.

Reply:
We definitely should rethink it, too many responsibilities for docs

However, as I see it will be include to production build? Can we use `ENV` condition?

Reply:
let me take a look on that..

Reply:
for prod we can remove this route, but it _might_ still be included (depends on config). I will add corresponding condition, like we are doing for prototype routes, and conclude on that